integration_test/third_party_apps_test/applications/dcgm/metadata.yaml

```
@@ -43,10 +43,10 @@ gpu_platforms: # p4, p100 don't emit DCGM profiling metrics
       - ubuntu-os-cloud:ubuntu-2004-lts
       - ubuntu-os-cloud:ubuntu-2204-lts
```

```suggestion
      - ubuntu-os-cloud:ubuntu-2204-lts
   - model: p100
    platforms:
      - ubuntu-os-cloud:ubuntu-2004-lts
   - model: p4
    platforms:
      - ubuntu-os-cloud:ubuntu-2004-lts
```
With the new receiver, P4 and P100 now can send DCGM metrics, so we should enable the DCGM test on those two GPUs.

---

Unfortunately, most profiling metrics are actually not available on P4 and P100, so I've had to revert this change.